docs/modules/ROOT/pages/type-definitions/schema-configuration/index.adoc

```diff
@@ -0,0 +1,37 @@
+[[type-definitions-schema-configuration]]
+= Schema Configuration
+
+Neo4j GraphQL Library supports several functionalities like CRUD operations, aggregation, filtering, and many others.
+
+To make them work, a large amount of GraphQL types are generated.
+
+For instance, starting from the type definitions:
+
+[source, graphql, indent=0]
+----
+type Movie { 
+    title: String
+    length: Int
+} 
+----
+
+
+In some cases, it may be advisable to reduce the scope of the API produced. Whatever the reason is to reduce the size of the schema, improve the performance, or limit access to unwanted operations, this section provides all the information needed to configure the API produced.
+
+From the type definitions above, let's say we want to prevent users from querying `movies` in the Neo4j Database. To do that, let's bring some changes to the initial type definitions:
```

Comment:
There are no type definitions above, but I suppose this was copied from the other file, right? Please let me know if that's the case so I can review this page again :)